lib/active_model/serializable_resource.rb

```diff
@@ -74,6 +75,14 @@ def serializer?
       use_adapter? && !!(serializer)
     end
 
+    def instrument
```

Comment:
I understand the reason to use `instrument` name here, but would be nicer if we could use something more meaningful mostly to help other ppl that might jumping into the code :smile:  maybe something like `notify_active_support` idk make yourself comfortable to come up if something better :grin: